src/store/wizardSlice.ts

```diff
@@ -37,6 +43,12 @@ type wizardState = {
     registrationType: string;
     activationKey: ActivationKeys['name'];
   };
+  profile: DistributionProfileItem | undefined;
+  openScap: {
+    profileId: OpenScap['profile_id'] | undefined;
+    profileName: OpenScap['profile_name'];
+    profileDescription: OpenScap['profile_description'];
+  };
```

Comment:
The wizardState is currently split by steps, `aws: {}` contains all values we need for AWS step, `registration: {}` contains everything needed for Registration step. It would be nice to have profile and openScap enclosed in a common object as well.

Also - I think it would make sense to rename `openScap` to `profileInformation` or something similar.

So the structure could then be something like this:
```typescript
openScap: {
    profile: DistributionProfileItem | undefined;
    profileInformation: {
      profileId: OpenScap['profile_id'] | undefined;
      profileName: OpenScap['profile_name'];
      profileDescription: OpenScap['profile_description'];
    };
}
```

What do you think?

Reply:
good idea